mlflow/langchain/runnables.py

```diff
@@ -91,9 +94,21 @@ def _load_model_from_path(path: str, model_config=None):
         return _load_base_lcs(path, model_config)
     if model_load_fn == _CONFIG_LOAD_KEY:
         return _load_model_from_config(path, model_config)
+    if model_load_fn == _PICKLE_LOAD_KEY:
+        return _load_from_pickle(os.path.join(path, model_config.get(_MODEL_DATA_KEY)))
     raise MlflowException(f"Unsupported model load key {model_load_fn}")
 
 
+def _validate_path(file_path: Union[str, Path]):
+    # Convert file to Path object.
+    load_path = Path(file_path)
```

Comment:
let's remove this comment. It's pretty clear that we're converting file to Path object.